src/components/Sidebar/Sidebar.jsx

```diff
@@ -31,13 +38,27 @@ const Sidebar = () => {
           showUserWrapper ? "user-wrapper user-wrapper-active" : "user-wrapper"
         }
       >
-        <h4>Do more with Blithe Brains</h4>
-        <p className="text-center gutter-bottom-8">
-          Log in in now to save videos, create playlists and much more.
-        </p>
-        <Link to="/login">
-          <button className="btn btn-solid-primary btn-rc">Log in</button>
-        </Link>
+        {isLoggedIn ? (
+          <>
+            <h4 className="text-center gutter-bottom-8">User's Name</h4>{" "}
+            <button
+              onClick={logoutHandler}
+              className="btn btn-solid-primary btn-rc"
+            >
+              Log out
+            </button>
+          </>
+        ) : (
+          <>
+            <h4>Do more with Blithe Brains</h4>
+            <p className="text-center gutter-bottom-8">
+              Log in in now to save videos, create playlists and much more.
+            </p>
+            <Link to="/login">
+              <button className="btn btn-solid-primary btn-rc">Log in</button>
```

Comment:
You can directly use navigate here from onClick as you would not have to nest a button inside a Link (anchor tag) or just apply the button classes to the Link.